shared/src/business/entities/cases/Case.ts

```diff
@@ -237,17 +237,10 @@ export class Case extends JoiValidationEntity {
   }
 
   static docketNumberSort(docketNumberA, docketNumberB) {
-    const aSplit = docketNumberA.split('-');
-    const bSplit = docketNumberB.split('-');
-
-    if (aSplit[1] !== bSplit[1]) {
-      // compare years if they aren't the same;
-      // compare as strings, because they *might* have suffix
-      return aSplit[1].localeCompare(bSplit[1]);
-    } else {
-      // compare index if years are the same, compare as integers
-      return +aSplit[0] - +bSplit[0];
-    }
```

Comment:
This leads to [20]95 and [19]95 being indistinguishable. I don't know if there is any value to the string comparison when we already have `getSortableDocketNumber`.

Reply:
@TomElliottFlexion: Do you see any issue with this change? `docketNumberSort` is used directly in `createCourtIssuedOrderPdfFromHtmlInteractor` and `getColdCases`, and indirectly in `getCasesForUserInteractor` and `getPractitionerCasesInteractor` (the code directly related to this story). I don't see any particular reason why these instances would need to use the `.localeCompare` string comparison rather than the `sortableDocketNumber`.

Reply:
Not at all! Looks like you just removed redundant code, thank you